semantic-python/src/Language/Python/Core.hs

```diff
@@ -92,20 +90,14 @@ toplevelCompile = flip compile (pure none)
 none :: (Member Core sig, Carrier sig t) => t Name
 none = unit
 
-locFromTSSpan :: SourcePath -> Span -> Loc
-locFromTSSpan fp = Data.Loc.Loc (rawPath fp)
-
 locate :: ( HasField "ann" syntax Span
           , CoreSyntax syn t
-          , Member (Reader SourcePath) sig
-          , Carrier sig m
+          , Applicative m
```

Comment:
1. The `Reader SourcePath` effect is now unused, but I haven’t removed the constraints on `compileCC` in order to  avoid conflicts with @patrickt’s ongoing work.

2. Likewise, the implementation is now pure, but I’ve left this in `Applicative` to avoid conflicts.

Reply:
Awesome. I’ll kickflip off of this and remove SourcePath.